lib/smart_answer/helpers/format_capture_helper.rb

```diff
@@ -0,0 +1,64 @@
+module SmartAnswer
+  module Helpers::FormatCaptureHelper
+    DEFAULT_FORMATS = {
+      govspeak: [/^body$/, /^post_body$/, /^next_steps$/],
+      text: [/^title$/, /^meta_description$/, /^hint$/, /^label$/, /^suffix_label$/, /^error_*./],
+    }.freeze
+
+    def content_for(name, content = nil, options = {}, &block)
+      if block_given?
+        options = content if content
+        content = capture(&block)
+
+        format = options.fetch(:format, default_format(name))
+        content = render_content(format, content)
+
+        block = nil
+      end
+
+      super(name, content, options, &block)
+    end
+
+  private
+
+    def default_format(name)
+      DEFAULT_FORMATS.each do |format, patterns|
+        return format if patterns.any? { |pattern| pattern.match?(name) }
+      end
+
+      :govspeak
+    end
+
+    def render_content(format, content)
+      case format
+      when :govspeak
+        render_govspeak(content)
+      when :text
+        render_text(content)
+      else
+        content
+      end
+    end
+
+    def render_govspeak(content)
+      content = strip_leading_spaces(content)
+      content = Govspeak::Document.new(content, sanitize: false).to_html
+      content = content.chomp.html_safe
```

Comment:
Can you help me understand why we need to `chomp` here? I think it's new.

Reply:
Ah I think the chomp was to remove newline between the content and govspeak tags from the component (line 50) i.e.
```html
<div class="govspeak-tag">
<p>content</p>
(chomp)
</div>
```